core-it-suite/pom.xml

```diff
@@ -159,6 +161,42 @@ under the License.
       </testResource>
     </testResources>
     <plugins>
+      <plugin>
+        <groupId>org.apache.maven.its</groupId>
+        <artifactId>core-it-bootstrap</artifactId>
+        <version>2.1-SNAPSHOT</version>
+        <executions>
+          <execution>
+            <id>boostrap</id>
+            <phase>process-test-classes</phase>
+            <goals>
+              <goal>download</goal>
+            </goals>
+          </execution>
+        </executions>
+        <configuration>
+          <file>src/test/resources/bootstrap.txt</file>
+          <dependencies>
+            <dependency>
+              <groupId>org.apache.maven.its.plugins</groupId>
+              <artifactId>maven-it-plugin-project-interpolation</artifactId>
+              <version>2.1-SNAPSHOT</version>
+              <scope>runtime</scope>
+              <exclusions>
+                <!--
+                NOTE: This artifact (and its dependency maven-verifier) are in use by the class loader employed by Surefire
+                to orchestrate this test. Re-downloading (newer snapshots of) these artifacts during the tests will wreck
+                havoc on the class loader, i.e. tests fail due to NoClassDefFoundError's in the test controller.
+                -->
+                <exclusion>
+                  <groupId>org.apache.maven.its</groupId>
+                  <artifactId>maven-it-helper</artifactId>
+                </exclusion>
+              </exclusions>
+            </dependency>
+          </dependencies>
+        </configuration>
```

Comment:
Fantastic config block

Reply:
This is a copy from one of the removed pom.  I haven't investigated if that's still necessary...